test/testvm.py

```diff
@@ -766,6 +823,7 @@ def _start_qemu(self, maintain=False, tty=False, monitor=None):
             self._locate_qemu_kvm(),
             "-m", str(MEMORY_MB),
             "-drive", "if=virtio,file=%s,index=0,serial=ROOT,snapshot=%s" % (drive_to_start, snapshot),
+            "-nographic",
```

Comment:
By adding this always, we never get a console with vm-run, right?  I think we should have a way to get into a VM even when network is hosed.